### PR TITLE
91 progress bar ui

### PR DESCRIPTION
**Pull request by ghost**
Kiaora, now have progress bar showing in the browser as a placeholder, not linked to any data yet

### CODE REVIEW

File: client/components/ProgressBar.jsx [hunk 1]
```diff
@@ -2,19 +2,20 @@ import React from 'react'
...
-  const divStyle = {
-    width: '50%',
-    margin: 'auto'
-  }
-  const percentage = props.placeHolderFromModule
+  const percentage1 = 10
+  const percentage2 = 20
+  const percentage3 = 30
+  const percentage4 = 15
+  // const percentage = props.placeHolderFromModule
```
Comment (emilyparkes) [inline client/components/ProgressBar.jsx]: remove comments

**Review by emilyparkes — CHANGES_REQUESTED** [review]

File: client/components/ProgressBar.jsx (hunk 1 above)
Comment (ghost) [reply client/components/ProgressBar.jsx]: Comment removed

File: client/components/ProgressBar.jsx [hunk 2]
```diff
@@ -2,19 +2,19 @@ import React from 'react'
...
 
   return (
     <React.Fragment>
-      <Bar style={divStyle}>
-        <Bar striped variant="success" now={{percentage}} key={1} />
-        <Bar variant="warning" now={{percentage}} key={2} />
-        <Bar striped variant="danger" now={{percentage}} key={3} />
-        <Bar striped variant="info" now={{percentage}} key={3} />
+      <h3>Cohort Progress</h3>
+      <Bar class='progressBar'>
```
Comment (emilyparkes) [inline client/components/ProgressBar.jsx]: should be `className` for react and `class` for html

**Review by emilyparkes — CHANGES_REQUESTED** [review]
And also please pull from dev

**Comment by ghost** [comment]
repulled branch and eliminated conflicting file

**Comment by emilyparkes** [comment]
Sorry please pull and solve conflicts again and I'll do your PR before any others :)

**Review by emilyparkes — APPROVED** [review]

**Pull request merged**